google/auth/external_account.py

```diff
@@ -203,6 +200,10 @@ def _constructor_args(self):
             args.pop("workforce_pool_user_project")
         return args
 
+    def lookup_trust_boundary(self, reqeust):
+        """Current implementation of inject dummy headers"""
```

Comment:
Can you rewrite this comment to describe what the function is actually supposed to do, and then have another sentence that explains currently it is just retrieving the default values.

Reply:
It is not clear what this function is for. Also there is a typo for `request`.